applications/vanilla/models/class.commentmodel.php

```diff
@@ -482,9 +482,10 @@ public function getDefaultLimit() {
      * @param int|null $limit Max number to get.
      * @param int $offset Number to skip.
      * @param string $order Order comments ascending (asc) or descending (desc) by ID.
+     * @param string $sort The column to sort by.
      * @return Gdn_DataSet SQL results.
      */
-    public function lookup(array $where = [], $permissionFilter = true, $limit = null, $offset = 0, $order = 'desc') {
+    public function lookup(array $where = [], $permissionFilter = true, $limit = null, $offset = 0, $order = 'desc', string $sort = 'CommentID') {
```

Comment:
Shouldn't the 'CommentID' be part of the `protected $_OrderBy` property?

Reply:
I believe the `protected $OrderBy` property is getting set in this method with the `$sort` and `$order` arguments on line 512, but perhaps I'm missing something?

Reply:
I think my last commit incorporates this correctly now.